operator-common/src/test/java/io/strimzi/operator/PlatformFeaturesAvailabilityTest.java

```diff
@@ -7,157 +7,155 @@
 
-    @BeforeAll
-    public static void before() {
-        vertx = Vertx.vertx();
+    private Supplier<List<String>> apis;
+    private Supplier<String> version;
+
+    private HttpServer mockHttp;
+
+    private final String defaultVersion =   "{\n" +
+                                            "  \"major\": \"1\",\n" +
+                                            "  \"minor\": \"16\",\n" +
+                                            "  \"gitVersion\": \"v1.9.1+a0ce1bc657\",\n" +
+                                            "  \"gitCommit\": \"a0ce1bc\",\n" +
+                                            "  \"gitTreeState\": \"clean\",\n" +
+                                            "  \"buildDate\": \"2018-06-24T01:54:00Z\",\n" +
+                                            "  \"goVersion\": \"go1.9\",\n" +
+                                            "  \"compiler\": \"gc\",\n" +
+                                            "  \"platform\": \"linux/amd64\"\n" +
+                                            "}";
+    private Vertx vertx;
+    private KubernetesClient kubernetesClient;
+
+    @BeforeEach
+    void setup(Vertx vertx) throws ExecutionException, InterruptedException {
+        this.vertx = vertx;
+
+        //Default to no apis.
+        apis = List::of;
+        version = () -> defaultVersion;
+
+        mockHttp = waitForMockApiServer(vertx);
+        kubernetesClient = new DefaultKubernetesClient("127.0.0.1:" + mockHttp.actualPort());
     }
 
-    @AfterAll
-    public static void after() {
-        vertx.close();
+    @AfterEach
+    void cleanup() throws ExecutionException, InterruptedException {
+        mockHttp.close().toCompletionStage().toCompletableFuture().get();
     }
 
+
     @Test
-    public void testVersionDetectionOpenShift(VertxTestContext context) throws InterruptedException {
-        String version = "{\n" +
-                "  \"major\": \"1\",\n" +
-                "  \"minor\": \"20\",\n" +
-                "  \"gitVersion\": \"v1.20.1\",\n" +
-                "  \"gitCommit\": \"c4d752765b3bbac2237bf87cf0b1c2e307844666\",\n" +
-                "  \"gitTreeState\": \"clean\",\n" +
-                "  \"buildDate\": \"2020-12-18T12:00:47Z\",\n" +
-                "  \"goVersion\": \"go1.15.5\",\n" +
-                "  \"compiler\": \"gc\",\n" +
-                "  \"platform\": \"linux/amd64\"\n" +
-                "}";
-
-        HttpServer mockHttp = startMockApi(context, version, Collections.EMPTY_LIST);
-
-        KubernetesClient client = new DefaultKubernetesClient("127.0.0.1:" + mockHttp.actualPort());
-
-        Checkpoint a = context.checkpoint();
-
-        PlatformFeaturesAvailability.create(vertx, client).onComplete(context.succeeding(pfa -> context.verify(() -> {
-            assertThat("Versions are not equal", pfa.getKubernetesVersion(), is(KubernetesVersion.V1_20));
-            stopMockApi(context, mockHttp);
-            a.flag();
-        })));
+    @Disabled
```

Comment:
Why is this disabled? Or maybe why i this present?

Reply:
Ah, that shouldn't have been committed.